docs/index.md

```diff
@@ -1043,6 +1045,16 @@ app/components
 </div>
 ```
 
+## Known issues
+
+### form_for compatiblilty
+
+ViewComponent is not currently compatible with `form_for`.
```

Comment:
I think `form_for` works mostly fine, it's the helpers that usual have problems when accepting blocks, or using the block provided form object in components.